src/nyc_trees/apps/event/models.py

```diff
@@ -100,6 +109,21 @@ class EventRegistration(NycModel, models.Model):
     class Meta:
         unique_together = ('user', 'event')
 
+    @classmethod
+    def my_events_now(cls, user):
+        """
+        Return events for which user has RSVPd but not checked in,
+        with times:  start-1h < now < end
+        """
+        now = timezone.now()
+        registrations = EventRegistration.objects \
+            .filter(user=user,
+                    did_attend=False,
```

Comment:
The event alert in the header goes away after you are checked-in. It might be nice to keep displaying the alert for the duration of the event.